prclz/cli.py

```diff
@@ -38,9 +40,9 @@ def split_geojson():
     pass 
 
 @prclz.command()
-@click.argument("--bldg_file", type=str, required=True, help="Path to master geojson file containing all building polygons")
-@click.argument("--gadm_path", type=str, required=True, help="Path to GADM file")
-@click.argument("--output_dir", type=str, required=True, help="Output directory for gadm-specific building files")
+@click.option("--bldg_file", type=str, required=True, help="Path to master geojson file containing all building polygons")
```

Comment:
is there a reason we swapped these to options?

Reply:
When I was having issues with click I was just testing out various possibilities bc I thought options was giving less trouble, so this is just vestigial basically